EnglishFilter/sections/foreign.txt

```diff
@@ -3598,6 +3598,8 @@ dispatch.co.kr##body > div[style="height: 45px;"]:not([id]):not([class])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38162
 yazaral11.com##div[id^="sitemap"] > a[href][target="_blank"]
 yazaral11.com###bitme
+! https://github.com/AdguardTeam/AdguardFilters/issues/39070
+||kongssew.com^$third-party
```

Comment:
better add adservers to `adservers.txt`, because they will not be added to SDN filter from `foreign.txt`